src/controllers/articleController.js

```diff
@@ -160,21 +161,29 @@ class ArticleManager {
    * @returns{object} an array of articles
    */
   static async listAllArticles(req, res) {
+    const pageNumber = paginate(req.query.page, req.query.pageSize);
     try {
       const articlesList = await Articles.findAll({
+        offset: pageNumber.offset,
+        limit: pageNumber.limit,
         include: [{
           as: 'author',
           model: Users,
           attributes: ['username', 'bio', 'image']
         }],
         attributes: ['id', 'slug', 'title', 'description', 'readtime', 'body', 'tagList', 'updatedAt', 'createdAt']
       });
+      if (!articlesList) {
+        return res.status(404).json({
+          error: 'there is no article'
+        });
+      }
       return res.status(200).json({
         articles: articlesList
       });
     } catch (error) {
-      return res.status(404).json({
-        error: 'there is no article'
+      return res.status(409).json({
```

Comment:
why did you use `conflict HTTP status code(409)` on `server error` ?

Reply:
Resolved